CHANGELOG.md

```diff
@@ -134,6 +134,7 @@ ___
 - Add NPM package-lock version check to CI (Manuel Trezza) [#7333](https://github.com/parse-community/parse-server/pull/7333)
 - Fix incorrect LiveQuery events triggered for multiple subscriptions on the same class with different events [#7341](https://github.com/parse-community/parse-server/pull/7341)
 - Fix select and excludeKey queries to properly accept JSON string arrays. Also allow nested fields in exclude (Corey Baker) [#7242](https://github.com/parse-community/parse-server/pull/7242)
+- Added safety check for attribute when filtering (Jason Posthuma) [#7421](https://github.com/parse-community/parse-server/pull/7421)
```

Comment:
Can you please make this more specific, for example:

> - Fix LiveQuery server crash when using `$all` query operator on a missing object key (Jason Posthuma) [#7421](https://github.com/parse-community/parse-server/pull/7421)

Reply:
Done